### DISCUSSION TITLE
Struggling to get off the ground with Smart Connections

### DISCUSSION THREAD

**Discussion by Uknowjuno** [brianpetro/obsidian-smart-connections#191] [Q&A]
Firstly. I actually asked Smart Connection how I can get it working and this is the answer I got... there's clearly a great opportunity to enter in some FAQ stuff into a shared embedding pool.. or something like that..? 

I have entered my API and restarted Obsidian. I exclude just my templates folder. But I'm having a really hard time getting smart connections to even recognise anything about my vault. 

I've now just force reset the embeddings... still no thing. 

Is there some general trouble shooting FAQ aroun ?

> To make smart connections work, you need to follow these steps:
> 
> Identify the devices you want to connect: Smart connections work by connecting different devices to each other. Identify the devices you want to connect and ensure that they are compatible with each other.
> 
> Install the necessary software: Some devices require specific software to be installed before they can be connected. Make sure you have installed the necessary software on each device.
> 
> Connect the devices: Once you have installed the necessary software, you can connect the devices. Follow the instructions provided by the manufacturer to connect the devices.
> 
> Configure the settings: Once the devices are connected, you may need to configure the settings to ensure that they work together seamlessly. This may involve adjusting the settings on each device or using a specific app to manage the connection.
> 
> Test the connection: Once everything is set up, test the connection to ensure that it is working correctly. If you encounter any issues, refer to the manufacturer's instructions or seek help from a technical support team.

**Comment by brianpetro** [comment]
Hi @Uknowjuno and thanks for trying Smart Connections.

> I actually asked Smart Connection how I can get it working and this is the answer I got... there's clearly a great opportunity to enter in some FAQ stuff into a shared embedding pool.. or something like that..?

That's a great idea/use-case!

Is it the "Smart View" or "Smart Chat" that your having trouble with?

Problems with the former often are from needing to setup billing in your OpenAI account.

If it's the chat, try using "my notes" in your chat input.

Let me know if that helps,
Brian 🌴
